package.json

```diff
@@ -27,5 +27,14 @@
   },
   "dependencies": {
     "@lingui/core": "^4.0.0"
+  },
+  "devDependencies": {
+    "@semantic-release/changelog": "^6.0.3",
+    "@semantic-release/commit-analyzer": "^9.0.2",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.7",
+    "@semantic-release/npm": "^10.0.3",
+    "@semantic-release/release-notes-generator": "^11.0.1",
+    "semantic-release": "^21.0.2"
```

Comment:
You actually only need `@semantic-release/changelog`, `@semantic-release/git` and `semantic-release`. The other semantic release dependencies can be removed.